deploy/k8s/charts/trustification/values.yaml

```diff
@@ -212,4 +213,13 @@ modules:
     image: {}
     rust: {}
 
+  report:
+    enabled: true
+    ingress: {}
+    image: {}
+    enabled_auth: false
```

Comment:
All other fields use `camelCase`. Is there a reason why to use `snake_case` here?